Snowflake.Data.Tests/UnitTests/Session/SFHttpClientPropertiesTest.cs

```diff
@@ -79,7 +79,7 @@ public void ShouldBuildHttpClientConfig()
             var config = properties.BuildHttpClientConfig();
 
             // then
-            Assert.AreEqual(properties.insecureMode, config.CrlCheckEnabled);
+            Assert.AreEqual(!properties.insecureMode, config.CrlCheckEnabled);
```

Comment:
one test run checks only true or false for this flag. Are we sure the other will succeed and is actually also in reverse of insecureMode?

Reply:
I added more tests.